build/teamcity-roachtest-stress.sh

```diff
@@ -3,19 +3,14 @@ set -euo pipefail
 
 source "$(dirname "${0}")/teamcity-support.sh"
 
-google_credentials="$GOOGLE_EPHEMERAL_CREDENTIALS"
+google_credentials="$GOOGLE_CREDENTIALS"
```

Comment:
```suggestion
google_credentials="$GOOGLE_ROACHSTRESS_CREDENTIALS"
```

This makes it possible to trace the credentials to being for the `cockroach-roachstress` project just by looking at the build config in TeamCity.